Keep downstream source builds independent of Go. Symbol prefixes are pregenerated
and audited when maintaining the native patches.

diff --git a/CMakeLists.txt b/CMakeLists.txt
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -633,19 +633,6 @@
 endif()
 
 
-if(BORINGSSL_PREFIX)
-  require_go()
-  add_custom_target(
-    verify_boringssl_prefix ALL
-    COMMENT "Verifying symbol prefixing"
-    COMMAND ${GO_EXECUTABLE}
-            run ${CMAKE_CURRENT_SOURCE_DIR}/util/audit_symbols.go
-            -ignore-symbols-with ${BORINGSSL_PREFIX}
-            $<TARGET_FILE:crypto>
-    DEPENDS crypto
-    USES_TERMINAL
-  )
-endif()
 
 if(INSTALL_ENABLED)
   install(TARGETS crypto ssl EXPORT OpenSSLTargets)
diff --git a/util/audit_symbols.go b/util/audit_symbols.go
--- a/util/audit_symbols.go
+++ b/util/audit_symbols.go
@@ -59,5 +59,6 @@ var skipWeakSymbols = []*regexp.Regexp{
 	regexp.MustCompile(`^__\w+\.get_pc_thunk\..*`),       // PIC
 	regexp.MustCompile(`^___asan_.*`),                    // AddressSanitizer
+	regexp.MustCompile(`^__clang_call_terminate$`),       // Clang exception handling
 	regexp.MustCompile(`^__cxa_.*`),                      // libc++abi
 	regexp.MustCompile(`^__dynamic_cast$`),               // libc++abi
 	regexp.MustCompile(`^__emutls_get_address$`),         // emulated TLS
